src/components/runtime-common.js

```diff
@@ -115,6 +115,7 @@ export const SaveFilesHelp = (isGalaxyDisk = false) => {
     p([
       'If you want to save some files permanently, such as input data, analysis outputs, or installed packages, ',
       h(Link, {
+        'aria-label': 'Save file help',
```

Comment:
Since this will be read as a sentence, 'move them to the workspace bucket' makes more sense to me than this label. If anything, we might insert the text 'learn how to' before the link, or something similar.
```suggestion
```